release.lua

```diff
@@ -0,0 +1,19 @@
+-- release.lua: => Arguments: [value]
+-- Release.lua deletes provided keys if all their values match the input. 
+
+-- Check all keys values matches provided input.
+local values = redis.call("mget", unpack(KEYS))
+for i, _ in ipairs(KEYS) do
+	if values[i] ~= ARGV[1] then
+		return false
+	end
+end
+
+-- Delete keys.
+for _, key in ipairs(KEYS) do
+	if redis.call("del", key) ~= 1 then
+		return redis.error_reply("redislock: failed to del " + key)
```

Comment:
this is a Go, not a Redis type error. you should wrap in Go code

Reply:
I simplified this part. We dont really need to check error for `del` and the loop above already ensure the key exists.